### PR TITLE
#160553067  add cart details

### PR DESCRIPTION
**Pull request by akpante3**
#### What does this PR do?
added more feedback about the add-to-cart action so that the user knows that the  action was accomplished 

#### Description of Task to be completed?
The page contains the following
1. the add to cart buttons now controls the add to cart options. 

#### How should this be manually tested?
https://akpante3.github.io/fast-food-fast/

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
#160553067

#### Screenshot
adjust

#### Questions:
N/A

### CODE REVIEW

**Comment by dogoyaro** [comment]
There should be a screenshot in this PR's description.

File: js/main.js [hunk 1]
```diff
@@ -2,18 +2,6 @@ const choose = Array.from(document.querySelectorAll('.choose'));
 // const addCart = Array.from(document.querySelectorAll('.add-cart'));
```
Comment (dogoyaro) [inline js/main.js]: remove commented out code.

**Review by dogoyaro — DISMISSED** [review]

File: js/main.js (hunk 1 above)
Comment (akpante3) [reply js/main.js]: @OsayamenOsaretin done

**Comment by akpante3** [comment]
> There should be a screenshot in this PR's description.
done

**Pull request merged**